site/dat/repo/various.json

```diff
@@ -554,7 +554,13 @@
     "versions": {
       "1.0.0": {
         "changes": "Initial version.",
-        "downloadUrl": "https://search.maven.org/remotecontent?filepath=io/github/rahulsinghai/jmeter.backendlistener.kafka/1.0.0/jmeter.backendlistener.kafka-1.0.0.jar",
```

Comment:
search.maven.org serves as redirect point for different regions. Why change it into fixed one?

Reply:
From my work network, search.maven.org was not working, so I thought to provide a direct link one. I will revert my changes. Thanks for the feedback. Closing this PR.